.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+<!-- TODO: Replace the title below -->
+# Your Title
+
+## Description
+
+<!-- TODO: Write a description of the proposed changes -->
+<!-- Remember to check the reminders! -->
+
```

Comment:
I noticed the bullet point added in the reminders, what I was hoping for is an empty field that the developer can fill in. 
For example, under the description, if you add `closes:`, the developer can then write something like `#5` to specify that the PR closes issue 5. This would make it clear where this information is and in some way required that every pull request is associated with an issue